presto-main/src/main/java/com/facebook/presto/sql/planner/optimizations/AggregationNodeUtil.java

```diff
@@ -0,0 +1,54 @@
+/*
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.facebook.presto.sql.planner.optimizations;
+
+import com.facebook.presto.spi.block.SortOrder;
+import com.facebook.presto.sql.planner.OrderingScheme;
+import com.facebook.presto.sql.planner.Symbol;
+import com.facebook.presto.sql.tree.Expression;
+import com.facebook.presto.sql.tree.SortItem;
+import com.facebook.presto.sql.tree.SymbolReference;
+
+import java.util.LinkedHashMap;
+import java.util.List;
+import java.util.function.Function;
+
+import static com.facebook.presto.sql.planner.PlannerUtils.toSortOrder;
+import static com.google.common.base.Preconditions.checkArgument;
+import static com.google.common.collect.ImmutableList.toImmutableList;
+
+public class AggregationNodeUtil
+{
+    private AggregationNodeUtil()
+    {}
+
+    public static OrderingScheme toOrderingScheme(List<SortItem> sortItems)
```

Comment:
I would extract `QueryPlanner::sort` to `PlannerUtils` rather than binding with aggregation node.